source/Calamari.Tests/AWS/S3Fixture.cs

```diff
@@ -49,7 +51,7 @@ private static JsonSerializerSettings GetEnrichedSerializerSettings()
 
         [Test]
         [Category(TestCategory.CompatibleOS.OnlyWindows)]
```

Comment:
@johnsimons do you have any knowledge on why these were marked to run on Windows only?

Reply:
Nope, I would expect this to work on all plats

Reply:
Tests are now running successfully on all plats in CI